### ISSUE TITLE
Feature request: Only send the last pressed key

### ISSUE THREAD

**Issue by darix** [zmkfirmware/zmk#2385]
To basically implement what can be seen here

https://www.youtube.com/watch?v=Feny5bs2JCg

**Comment by caksoylar** [comment]
I think this is related: https://github.com/qmk/qmk_firmware/pull/24000

Warning to readers: The code is GPL, see the [clean room policy](https://zmk.dev/docs/development/clean-room). I linked because the PR description/docs and naming can be useful.

**Comment by darix** [comment]
yeah but 24000 only implements part of it. the resuming previous key part is not in this pull request. [see this comment and the reply](https://github.com/qmk/qmk_firmware/pull/24000#issuecomment-2241749830)

**Comment by kovasap** [comment]
I have a suspicion this is already possible with zmk's existing behaviors, but I don't know how.  Either way, I'm also interested in this feature!
